src/components/ReservationList.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchReservationsSuccess } from '../redux/reservationsReducer';
+import ReservationCard from './ReservationCard';
+import '../Reservations.css';
+import { request } from '../redux/users/api';
+
+const ReservationList = () => {
+  const dispatch = useDispatch();
+  const reservationList = useSelector((state) => state.reservations);
+  const reservations = [];
+  const user = JSON.parse(localStorage.getItem('user'));
+  console.log(user);
```

Comment:
- [ ] Please, remove this old debugging `console.log` statement in order to improve the cleanliness of the repo.